packages/web/src/components/spa/HackAUBG/LandingAnimation/landing_animation.css

```diff
@@ -56,7 +60,7 @@
     }
     .animation-container {
         padding-top: 6vh;
-        height: 45vh;
+        height: 100vh;
```

Comment:
If you make the height: 100vh in the media query max-width: 1300px, you dont have to do the same for the other media queries since this attribute already applies to all of them (because they are lower than 1300px)

Reply:
we don't need any media queries for setting height at all if the height is always 100vh